IntelliTect.TestTools.WindowsTestWrapper/ExampleProjects/Notepad.AutomatedTests/Helpers/NotepadWindowHelpers.cs

```diff
@@ -10,6 +12,37 @@ public class NotepadWindowHelpers
         private readonly NotepadWindow _NotepadWindow = new NotepadWindow();
         private readonly SaveAsWindow _SaveAsWindow = new SaveAsWindow();
 
+        public bool ValidateTextFile(string location, string fileName, string expectedText)
+        {
+            bool comparisonPass = false;
+            string combinedPath = Path.Combine( location, fileName );
+            if ( combinedPath == expectedText )
+            {
+                comparisonPass = true;
+            }
+            return comparisonPass;
+        }
+
+        public void DeleteDocument(string location, string fileName)
+        {
+            var combinedPath = Path.Combine(location, fileName);
+            int retryAttempts = 0;
+            do
+            {
+                File.Delete(combinedPath);
+                if (!File.Exists(combinedPath))
+                {
+                    break;
+                }
+                Thread.Sleep(100);
+                retryAttempts++;
+            } while (retryAttempts < 10);
+            if (File.Exists(combinedPath))
```

Comment:
That is ridiculously helpful. I'll do some more updates based on the information and the code. Am I correct in assuming that if I do:
Try{ //action }
catch (UnauthorizedAccessException) { //retry stuff}
catch (Exception) {throw;}

Then the last catch statement will basically catch and throw everything else that isn't covered by the first one (or, first two, once I have one for catching the IOException)?